jwst/associations/lib/asn_schema_jw_level2b.json

```diff
@@ -1,5 +1,5 @@
 {
-    "$schema": "http://json-schema.org/schema#",
+    "$schema": "http://json-schema.org/draft-04/schema",
```

Comment:
So we'll have to manually update this whenever we want to start using a new draft?

Reply:
Yes that's correct.  Incidentally, later versions aren't backwards compatible with draft-04, so we'll need to audit these files anyway.